lib/application.ts

```diff
@@ -1021,19 +1021,22 @@ export class SNApplication {
       const syncPromise = this.syncService!.sync({
         mode: SyncModes.DownloadFirst,
         checkIntegrity: true,
-        queueStrategy: SyncQueueStrategy.ForceSpawnNew
+        queueStrategy: SyncQueueStrategy.ForceSpawnNew,
+        awaitAll: true,
```

Comment:
I thought this should be `awaitAll: awaitSync,`? You don't want to awaitAll on regular sign in because it will completely block the UI until all X user notes are downloaded (could be 3000).